desktop/apps/auction/routes.js

```diff
@@ -48,11 +49,31 @@ export async function index (req, res, next) {
     // TODO: Refactor out Backbone
     const auctionModel = new Auction(sale)
 
+    /**
+     * An e-commerce sale is one which is created by following these steps:
+     * https://github.com/artsy/auctions/blob/master/auctions_support_faq.md#how-do-i-create-an-e-commerce-sale
+     * It enables relations between one or more conventional auctions and
+     * represents "Buy Now"-like artworks
+     */
+    const isEcommerceSale = !sale.is_auction && isObject(sale.promoted_sale)
```

Comment:
Didn't notice this before, but the `promoted_sale` association only matters for showing the rail on an auction page. For the generic sale page, just checking for `is_auction` is sufficient.

Reply:
I would also update your description-- an e-commerce sale is just a collection of artworks that can be sold over a fixed period of time (but via e-commerce or other methods instead of through bidding). The `promoted_sale` relation enables a rail of works from a separate sale to be shown on a different sale page.

It's def a confusing thing-- this just feels like it's conflating the two concepts 😅 .